### PR TITLE
RMMIS-6176: Created Accordion component and accompanying test

### PR DESCRIPTION
**Pull request by kjschulz**

### CODE REVIEW

File: src/Accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  handleSelection(eventKey) {
+    this.setState({defaultActiveKey: eventKey});
+  }
+
+  getActiveStyle(idx, defaultActiveKey) {
+    return (idx === defaultActiveKey) ? 'info' : 'default';
+  }
+
+  render() {
+    console.log('state', this.state);
```
Comment (mallenscdhhs) [inline src/Accordion.jsx]: (areyoukiddingme)

**Comment by kjschulz** [comment]
Guys, this options-mixin-spec is some crazy :hankey: !!!  I know it's only a short-term fix but I renamed it with a prepended "_" to avoid the weirdness.  It actually doesn't seem to have anything to do with my test but more the order the tests appear in.  I could use a lil help and am open to suggestion if anybody has any ideas.

File: src/Accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/* jshint node:true */
```
Comment (mallenscdhhs) [inline src/Accordion.jsx]: (feelsbadman)

File: src/Accordion.jsx (hunk 1 above)
Comment (kjschulz) [inline src/Accordion.jsx]: (doh), removing

**Pull request merged**